integration-tests/__tests__/__snapshots__/coverage_report.test.js.snap

```diff
@@ -27,8 +27,8 @@ exports[`collects coverage only from specified files avoiding dependencies 1`] =
 "----------|----------|----------|----------|----------|----------------|
 File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
 ----------|----------|----------|----------|----------|----------------|
-All files |    85.71 |      100 |       50 |    85.71 |                |
- sum.js   |    85.71 |      100 |       50 |    85.71 |             12 |
+All files |       75 |      100 |       50 |       75 |                |
```

Comment:
is this supposed to happen?

Reply:
Yes. I manually checked the coverage folder for HTML report, there was 2 arrow function declaration statement with one statement inside each. And only 1 function ran in the test case. So that is 3/4 or 75%

Reply:
Let me look into why the CI broke.

Reply:
@SimenB Can you rerun the failed CI build?